.gitignore

```diff
@@ -0,0 +1,2 @@
+# System files.
+.DS_store
```

Comment:
Go to your repo on the Raspberry Pi and run `git status`. Provide me the list. Some of them need to be ignored by `git` so we will want to add them here.